js/js.tests/build.gradle.kts

```diff
@@ -2,6 +2,10 @@ import com.moowork.gradle.node.NodeExtension
 import com.moowork.gradle.node.npm.NpmTask
 import org.gradle.internal.os.OperatingSystem
 
+tasks.withType<Test> {
```

Comment:
Could we turn it for JS off for now?

Reply:
I would rather add a workaround if possible. If it's still broken after the latest changes I'll get rid of the parallelization for the JS build.